files/image_config/sysctl/sysctl-net.conf

```diff
@@ -14,7 +14,7 @@ net.ipv4.conf.all.arp_accept=0
 net.ipv4.conf.all.arp_announce=1
 net.ipv4.conf.all.arp_filter=0
 net.ipv4.conf.all.arp_notify=1
-net.ipv4.conf.all.arp_ignore=2
+net.ipv4.conf.all.arp_ignore=0
```

Comment:
I'm trying to investigate and find out why this was originally set to 2, will come back once I have an answer

Reply:
This would make the Loopback IP reachable from any connected device, including servers/blades configured in a Vlan. I@prsunny @lguohan is this acceptable? I can't think of any other impact this might have. Since this change is making `arp_ignore` more permissive, existing ARP functionality should hopefully be unaffected.